pro/src/new_components/VenueForm/Informations/Informations.tsx

```diff
@@ -1,14 +1,90 @@
-import React from 'react'
+import { useFormikContext } from 'formik'
+import React, { useState } from 'react'
 
 import FormLayout from 'new_components/FormLayout'
-import { TextInput } from 'ui-kit'
+import { Select, TextArea, TextInput } from 'ui-kit'
+
+import { IVenueFormValues } from '..'
+
+import SiretOrCommentFields from './SiretOrCommentFields'
+
+export interface IInformations {
+  isCreatedEntity: boolean
+  readOnly: boolean
+  updateIsSiretValued: (value: boolean) => void
+  venueIsVirtual: boolean
+  venueTypes: SelectOption[]
+  venueLabels: SelectOption[]
+}
+
+const Informations = ({
+  isCreatedEntity,
+  readOnly,
+  updateIsSiretValued,
+  venueIsVirtual,
+  venueTypes,
+  venueLabels,
+}: IInformations) => {
+  const { initialValues } = useFormikContext<IVenueFormValues>()
+  const [isFieldNameFrozen, setIsFieldNameFrozen] = useState(false)
+  const siretLabel = isCreatedEntity
+    ? 'SIRET du lieu qui accueille vos offres (si applicable) :'
+    : 'SIRET :'
 
-const Informations = () => {
   return (
     <>
       <FormLayout.Section title="Informations lieu">
         <FormLayout.Row>
-          <TextInput name="publicName" label="Nom d’usage du lieu:" />
+          {!venueIsVirtual && (
+            <SiretOrCommentFields
+              siretLabel={siretLabel}
+              readOnly={readOnly || !!initialValues.siret}
+              isToggleDisabled={!isCreatedEntity}
+              isCreatedEntity={isCreatedEntity}
+              setIsFieldNameFrozen={setIsFieldNameFrozen}
+              updateIsSiretValued={updateIsSiretValued}
+            />
+          )}
+        </FormLayout.Row>
+        <FormLayout.Row>
+          <TextInput
+            name="name"
+            label="Nom du lieu :"
+            readOnly={isFieldNameFrozen}
+          />
+        </FormLayout.Row>
+        <FormLayout.Row>
+          <TextInput name="publicName" label="Nom d’usage du lieu :" />
+        </FormLayout.Row>
+        <FormLayout.Row>
+          <TextInput name="mail" label="Mail :" type="email" />
```

Comment:
```suggestion
          <TextInput name="mail" label="Mail :" type="email" placeholder="email@exemple.com" />
```
et pareil, on a rajouté des placeholder sur tous les champs email et url